app/src/test/java/com/kickstarter/viewmodels/UpdateViewModelTest.java

```diff
@@ -21,17 +22,46 @@
 import rx.observers.TestSubscriber;
 
 public final class UpdateViewModelTest extends KSRobolectricTestCase {
+  private final Intent defaultIntent = new Intent()
```

Comment:
something new I'm trying out here--maybe it's nice to have a default `intent` in our view model tests. this eliminates the need to repeatedly create new intents (e.g. `vm.intent(new Intent()....)`) when starting a test that doesn't need to keep a reference to the intent objects

it would also give us an immediate reminder / some guidance as to what our view model's intent is and should be initialized with.